res/TemplateBatchFiles/SelectionAnalyses/FUBAR.bf

```diff
@@ -48,6 +48,9 @@ fubar.json = {
                     terms.fubar.posterior  :      {}
                 };
 
+fubar.display_orders = {terms.original_name: -1,
```

Comment:
Starting from -1 seems curious.

Reply:
The original names are the only attributes that should never be looped over or referenced for model fits, so I put -1 as an identifying safeguard.